state/human.context.js

```diff
@@ -70,7 +83,7 @@ function HumanProvider(props) {
   }
 
   const getTokensBalance = async () => {
-    if (!human) return
+    if (!humanSDK.isReady()) return
```

Comment:
same here
the thing is to be sure the async state of the client (PoC) has been properly updated